pandas/tests/test_resample.py

```diff
@@ -985,6 +986,19 @@ def test_resample_basic(self):
         expect = s.groupby(grouper).agg(lambda x: x[-1])
         assert_series_equal(result, expect)
 
+    def test_resample_string_kwargs(self):
+        rng = date_range('1/1/2000 00:00:00', '1/1/2000 00:13:00', freq='min',
+                         name='index')
+        s = Series(np.random.randn(14), index=rng)
+
+        # Check that wrong keyword argument strings raise an error
+        with pytest.raises(ValueError) as e_info:
```

Comment:
@jreback There is also a linting error here because of the unused `e_info`. I will also remove these, okay?